subprojects/logging/src/main/java/org/gradle/internal/featurelifecycle/LoggingDeprecatedFeatureHandler.java

```diff
@@ -150,6 +159,17 @@ private static boolean isGradleScriptElement(StackTraceElement element) {
         return false;
     }
 
+    private static boolean isSystemElement(StackTraceElement element) {
+        String className = element.getClassName();
+        return className.startsWith("org.codehaus.groovy.")
```

Comment:
This is very brittle, I've already found things missing, just by looking at `DeprecationMessagesTest`. If we were to add this, it would be very easy to break by unrelated future changes.